infrastructure/docker/celo/start.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+mkdir -p /data/keystore
```

Comment:
Would it be worth defining `/data` dir as an argument or env variable? The script would be a little more generic and maybe could be used outside of this specific Docker image.

Reply:
Done. I defined `CELO_DATA_DIR` and `CELO_INIT_DIR`. Please see 9f868b7.